qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/CheckModulesTest.java

```diff
@@ -1,44 +0,0 @@
-/**
- * This Source Code Form is subject to the terms of the Mozilla Public License,
- * v. 2.0. If a copy of the MPL was not distributed with this file, You can
- * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
- * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
- *
- * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
- * graphic logo is a trademark of OpenMRS Inc.
- */
-package org.openmrs.contrib.qaframework.legacy;
-
-import static org.junit.Assert.assertFalse;
-
-import java.util.List;
-
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.openmrs.contrib.qaframework.helper.BuildTests;
```

Comment:
Are these deletions related to the ticket at hand?

Reply:
Yes they are part of the test we are getting rid of the whole class.

Reply:
Can you explain how it is related to User Role Based Login?

Reply:
It is not related to user Role Based Login , However since it checks which module gets started/Not started  with in the server, it would be good deal to also include it here, However am of no objection to remove it. cc @kdaud  do you have suggestion on this.

Reply:
Do you mind taking a look at this? https://opensource.com/article/18/6/anatomy-perfect-pull-request

Reply:
Sure thanks. Let me correct it .